WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -81,9 +81,35 @@ msgctxt "app_store_keywords"
+"\n"
+"We reorganized your Stats screen to show new information about traffic and insights. We also added a Subscribers tab so you can better analyze information about your site subscribers. Please like and subscribe.\n"
+"\n"
+"We fixed multiple issues with post syncing, like posts not loading when the app opens and not showing up in the right status tab. A place for every post, and every post in its place.\n"
+"\n"
+"When you’re editing a page, you can now set the parent page within the page’s settings. We added slug and excerpt fields to the settings area, and we removed the status field. We also improved the way different settings are ordered and grouped.\n"
+"\n"
+"We also resolved issues with the Publish Date field in your post settings. The new date picker makes it easy to choose your publication date, even if you’re in a different time zone from the blog. You’ll only see the Publish Date field when you’re ready to hit publish.\n"
+"\n"
+"We added a new Media Uploads screen to your publishing workflow so you can see uploads in progress. You’ll see any error messages for failed uploads here, along with the option to cancel them. Oh, and speaking of errors, we improved notifications for errors with uploading and publishing.\n"
+"\n"
+"We updated the “more” menu in the editor with a new look and feel, and we changed the design for your revisions list. We also added a badge to indicate when a post is password protected. (We don’t recommend using the password 1-2-3-4-5. Amazingly, that’s the same combination we have on our luggage.)\n"
+"\n"
+"Pending posts now have clearer workflows for admins and contributors. We also updated the success view for published posts. It’s now easier to Share and Blaze a post.\n"
+"\n"
+"Changed your mind about trashing a post? You can now view and restore trashed posts and pages from the editor.\n"
+"\n"
+"We removed the “draft uploaded” bar that appeared on the bottom of your screen when you were saving or updating a draft post. This did not spark joy, so we thanked it and said goodbye.\n"
+"\n"
+"Finally, we squashed a long list of bugs and fixed a handful of rare crashes caused by code issues.\n"
+"\n"
+"- If a draft on your device is different from the draft saved to the web, the app will now show a conflict resolution screen instead of making changes on its own.\n"
+"- Discarding changes to a draft will no longer delete the entire draft.\n"
+"- The footer will show the correct date when a draft was created.\n"
+"- You’ll now see an error message when you assign a page author who isn’t eligible to be an author.\n"
+"- Switching between tabs in Posts will show all your draft posts and published content, no matter how much of it you have. Yes, even you, the user with 1,239 saved drafts. We’re impressed and slightly concerned, but we respect your writing process.\n"
```

Comment:
This version fixes an issue where some posts disappear from their respective tabs. 

> Switching between tabs in Posts will show all your draft posts and published content, no matter how much of it you have.

This is not the case. I'd suggest removing this line.

cc @salimbraksa